indico/MaKaC/PDFinterface/conference.py

```diff
@@ -1761,10 +1761,16 @@ def __init__(self, conf, badgeTemplate, marginTop, marginBottom, marginLeft, mar
         self.__marginColumns = marginColumns
         self.__marginRows = marginRows
         if registrantList == 'all':
-            self.__registrantList = Registration.find(~Registration.is_deleted, Registration.event_id == conf.id).all()
+            self.__registrantList = (Registration
+                                     .find(~Registration.is_deleted, Registration.event_id == conf.id)
+                                     .order_by(Registration.last_name)
```

Comment:
Actually, I think we sort like this in a few places now.
Maybe something like this on the `Registration` model wouldn't be too bad:
```
@classproperty
@classmethod
def order_by_name(cls):
    return db.func.lower(Registration.last_name), db.func.lower(Registration.first_name), Registration.friendly_id
```

And then `.order_by(*Registration.order_by_name)`